### ISSUE TITLE
constrain_start_time, constrain_end_time not working between sunset and sunrise

### ISSUE THREAD

**Issue by bbrendon** [AppDaemon/appdaemon#1690]
### What happened?

Updated, and porch light stopped working. (tested at night)
Updated to latest dev and tested. Still not working. (tested at night)

If I comment out the constrain_* lines below, it works.

```
FrontDoorLight:
  class: MotionLights3
  module: motion_lights3
  delay: 900
  entity_off: light.upb_livingrm_porch
  entity_on: light.upb_livingrm_porch
  sensors:
    - binary_sensor.front_door
  constrain_start_time: sunset
  constrain_end_time: sunrise
```

Anyone else notice this?

### Version

dev

### Installation type

Docker container

**Comment by acockburn** [comment]
OK, looking into this, I've built.a test app now I need to wait until sunset ...

**Comment by acockburn** [comment]
Fixed in dev, will be in 4.3.0
